src/callbacks.c

```diff
@@ -307,16 +307,15 @@ G_MODULE_EXPORT void on_redo1_activate(GtkMenuItem *menuitem, gpointer user_data
 
 G_MODULE_EXPORT void on_cut1_activate(GtkMenuItem *menuitem, gpointer user_data)
 {
-	GeanyDocument *doc = document_get_current();
 	GtkWidget *focusw = gtk_window_get_focus(GTK_WINDOW(main_widgets.window));
 
-	if (GTK_IS_EDITABLE(focusw))
+	if (GTK_IS_EDITABLE(focusw)) {
```

Comment:
we don't have a strict rule on whether to have brackets or not on single statements blocks, but we definitely put the brackets on their own lines :)

And regarding your references, don't reproduce the `else if` with each keyword on separate lines, I don't know why it's like it there but we don't want that to spread (e.g. that part in what you changed is good).